components/dashboard/src/index.css

```diff
@@ -55,6 +55,11 @@
     button {
         @apply cursor-pointer px-4 py-2 my-auto bg-green-600 dark:bg-green-700 hover:bg-green-700 dark:hover:bg-green-600 text-gray-100 dark:text-green-100 text-sm font-medium rounded-md focus:outline-none focus:ring transition ease-in-out;
     }
+    button.reset {
```

Comment:
This class will be useful in scenarios like this where we use buttons purely for semantics and a11y